src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -121,4 +241,115 @@ export class OsdListComponent implements OnInit {
   configureClusterAction() {
     this.bsModalRef = this.modalService.show(OsdFlagsModalComponent, {});
   }
+
+  markOut() {
```

Comment:
This is already just boilerplate code and I tend to not change it as it's very easy to react to changes. Having this generalized makes it less flexible to changes and I think these modals are still subject to change. It would also kind of do things implicit instead of explicit. I'm not against this, but maybe now is not the time.

Reply:
This is already just boilerplate code and I tend to not change it now as it's very easy to react to changes. Having this generalized makes it less flexible to changes and I think these modals are still subject to change. It would also kind of do things implicit instead of explicit. I'm not against this, but maybe now is not the time.

Reply:
As the same code is repeated three times in the template and the controller, I suggest clean code over redundant code.
Do you await that huge changes to the modal of these actions? Not only a description change?